src/styles/components/_browser-compat.scss

```diff
@@ -0,0 +1,49 @@
+.browser-compat {
+
+  .browser-compat__icon {
+    height: 48px;
+    width: 48px;
+    margin: 6px;
+    display: inline-block;
+    position: relative;
+  }
+
+  .browser-compat__icon::before {
+    content: attr(data-version);
+    display: block;
+    width: 18px;
+    line-height: 18px;
+    height: 18px;
+    border-radius: 50%;
+    bottom: 0;
+    right: 0;
+    position: absolute;
+    text-align: center;
+    color: #fff;
+    font-size: 12px;
+  }
+
+  .browser-compat__icon.browser-compat--false::before {
+    background: red;
+  }
+
+  .browser-compat__icon.browser-compat--true::before {
+    background: green;
+  }
+
+  .browser-compat--chrome {
+    background: content-box url('/images/icons/chrome.svg');
```

Comment:
My advice here is set `background: content-box no-repeat transparent` on a shared element (since you can use shorthand, but _have_ to have something), and then each 'type' becomes simpler, and you can just set `background-image: url(....)` on each.